rust/pact_ffi/src/verifier/mod.rs

```diff
@@ -134,7 +134,7 @@ ffi_fn! {
     /// All string fields must contain valid UTF-8. Invalid UTF-8
     /// will be replaced with U+FFFD REPLACEMENT CHARACTER.
     ///
-    fn pactffi_verifier_update_provider_state(
+    fn pactffi_verifier_set_provider_state(
```

Comment:
It's my mistake. It should follow naming pattern from `pactffi_verifier_set_provider_info`